draft-tiruben-reverse-http.md

```diff
@@ -179,13 +179,13 @@ Reverse HTTP is principally intended for use between intermediaries and origins.
 
 ## Stolen key attacks {#stolen-keys}
 
-As noted in {{Section 4 of !RFC8336}}, accepting ORIGIN frames without DNS confirmation facilitates the use of stolen keys, and thus increases the incentive to steal these keys.  The mitigations listed in that section also apply here, and are RECOMMENDED.  Intermediaries also MAY impose a DNS confirmation requirement, although this weakens the DoS defense provided by Reverse HTTP ({{ip-leaks}}).
+As noted in {{Section 4 of !RFC8336}}, accepting ORIGIN frames without DNS confirmation facilitates the use of stolen keys, and thus increases the incentive to steal these keys.  The mitigations listed in that section also apply here, and are RECOMMENDED.  Intermediaries also MAY impose a DNS confirmation requirement, although this weakens the DDoS defense provided by Reverse HTTP ({{ip-leaks}}).
```

Comment:
I'm not sure we need to restrict this to "DDoS".  For example, a volumetric attack using IP spoofing and reflection-amplification is not necessarily a DDoS attack; it could originate from a single source.

Reply:
Yes, we can update it to "DoS/DDoS".